presto-main/src/test/java/com/facebook/presto/operator/scalar/BenchmarkArrayDistinct.java

```diff
@@ -86,14 +83,7 @@
     public Object arrayDistinct(BenchmarkData data)
             throws Throwable
     {
-        int position = 0;
-        List<Page> pages = new ArrayList<>();
-        while (position < data.getPage().getPositionCount()) {
-            position = data.getPageProcessor().process(SESSION, data.getPage(), position, data.getPage().getPositionCount(), data.getPageBuilder());
-            pages.add(data.getPageBuilder().build());
-            data.getPageBuilder().reset();
-        }
-        return pages;
+        return data.getPageProcessor().processColumnar(SESSION, data.getPage(), TYPES);
```

Comment:
Are there benchmarks results somewhere which shows how performance changes between three stages:

1. Original non-columnar benchmarks
1. Columnar benchmarks on old codebase
1. Columnar benchmarks with your changes?

It would be nice to have it is as part of one of the commit messages.